Feat/japan chair map

jdromero88 opened the pull request.

smaraghi commented on japan-edp-map/css/world.css:
```diff
@@ -0,0 +1,163 @@
+  .open {
+    right: 0%
+  }
+
```
Remove spacing

smaraghi reviewed (changes requested):
Jose, you did an amazing job with this map! Just a couple of notes below:

- I would change the names of the files to map.css and map.js (Make sure you change them in the index.html as well). These were the names of the Americas map files. We should update them
- I would change your handling of financial support and source. There is a chance that the term 'source' can be added at a later date into the description text so doing indexOf('source') is a bit risky and not a fool proof solution. The better option is to create a new column in the database and add source there separately. 

You will then need to update the codebase to the following:

`  if (!sidePanel.isOpen()) {
    var data = event.data;
    var content = '';

    content += `
    <h2 class="sidePanelHeaderStyle">
      ${data.country}
    </h2>
    <p class="side-panel-value">Description: <span>${data.description}</span> </p>
    <p class="side-panel-value">Financial Support: <span>${data.financial_support}</span> </p>
    `
    if (data.source) {
     content += `<p class="side-panel-value">Financial Support: <span>${data.source}</span> </p>`
    }
    content +=
    `
    <p class="side-panel-value">Focus Areas: <span>${data.focus_areas}</span> </p>
    <p class="side-panel-value">Major Recipients: <span>${data.major_recipients}</span> </p>
    <p class="side-panel-link">For more details, click <a href="https://docs.google.com/spreadsheets/d/e/2PACX-1vTxYNVcd_DsRaJzHo6c3dN78Y9uOypX2jX4VzEaJZpgX_t9qXFfCzNENobD7aFXB-HeVawqjFXtslKI/pubhtml" target="_blank">here</a>.</p>
    `;
    panelContent.innerHTML = content;
  }`
  
  You can see that there is also a null check for data.source. After in your css you will want to add some Margin for spacing and your styling. Your styles should be applying as long as your class name does not change. 
  
  The addition of the spans you added will no longer be necessary which will also clean up the HTML. 
  
  - Close Button: Should be on the same level with the title and to the left more (above the line)
  
  Thank you for your first go on this!

jdromero88 commented:
Hey @smaraghi I added the ```z-index: 1;``` on the class ```.close-btn```. Because when I moved it to be within the line it started going behind the div ```panel-content``` so it wasn't working.

smaraghi commented on japan-edp-map/index.html:
```diff
@@ -0,0 +1,76 @@
+
+    <section class="toolbox">
+      <header>
+        <h1 class="box-header">Enhancing Democratic Partnership in the <br> Indo-Pacific Region</h1>
```
There is a br tag in the middle of this h1 tag

jdromero88 replied:
Oh yes! thank for catching this! I added that because the word ```Indo-Pacific```  was breaking on the ```-``` and it looked weird.
Anyways I added a span tag to get ride of the br tag.

smaraghi commented on japan-edp-map/css/map.css:
```diff
@@ -0,0 +1,191 @@
+
+/* Override leaflet default styles */
+.leaflet-top {
+  position: relative !important
```
Can you add spaces after each of these blocks? Its good to get in the habit of doing this as our linters will catch these mistakes in other projects.

jdromero88 replied:
you got it @smaraghi !!

smaraghi reviewed (changes requested):
Jose this looks amazing! 

I think as far as the CSS goes - we should try to have the margin and padding set in rems as thats better for the browser.
